### PR TITLE
Hopefully fixed mistakes in quoting

### PR DESCRIPTION
**Pull request by AaronNGray**
**What does this PR do?**

Hopefully fix some mistakes in quoting and a cropped line ending.

Thanks for the contribution! Please provide a concise description of the problem this request solves.

**How does this PR change Premake's behavior?**

Its just the documentation.

**Anything else we should know?**

**Did you check all the boxes?**

- [x] Focus on a single fix or feature; remove any unrelated formatting or code changes
- [ ] Add unit tests showing fix or feature works; all tests pass
- [ ] Mention any [related issues](https://github.com/premake/premake-core/issues) (put `closes #XXXX` in comment to auto-close issue when PR is merged)
- [ ] Follow our [coding conventions](https://github.com/premake/premake-core/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] Minimize the number of commits
- [ ] Align [documentation](https://github.com/premake/premake-core/tree/master/website) to your changes

*You can now [support Premake on our OpenCollective](https://opencollective.com/premake). Your contributions help us spend more time responding to requests like these!*

### CODE REVIEW

**Comment by AaronNGray** [comment]
Documentation fixes just need checking.

File: website/docs/Custom-Rules.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ rule "MyCustomRule"
 
   buildmessage 'Compiling %(Filename) with MyCustomCC'
   buildcommands 'MyCustomCC.exe -c "%(FullPath)" -o "%(IntDir)/%(Filename).obj"'
-  buildoutputs '%(IntDir)/%(Filename).obj"'
+  buildoutputs '%(IntDir)/%(Filename).obj'
```
Comment (nickclark2016) [inline website/docs/Custom-Rules.md:23]: Should we insert a quote before intdir rather than removing the trailing?

Comment (AaronNGray) [reply website/docs/Custom-Rules.md:23]: I ruled that out :- https://github.com/premake/premake-core/discussions/1935

File: website/docs/Custom-Rules.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ rule "MyCustomRule"
 
   propertydefinition {
     name = "StripDebugInfo",
-    ind = "boole
+    ind = "boolean",
```
Comment (thomashope) [inline website/docs/Custom-Rules.md:48]: I believe this should also be `kind` rather than `ind`?

Comment (nickclark2016) [reply website/docs/Custom-Rules.md:48]: I think so as well.  Not 100% sure what happened to this file, but it appears that it got messed up.

**Review by nickclark2016 — CHANGES_REQUESTED** [review]
Change `ind` to `kind`.

**Comment by AaronNGray** [comment]
@nickclark2016 Yes probably better to correct it at your end as one commit

**Comment by nickclark2016** [comment]
> @nickclark2016 Yes probably better to correct it at your end as one commit

Wait, did I just get roped into fixing this?

**Pull request closed without merging**